core/workflow-operator/src/main/scala/edu/uci/ics/amber/operator/visualization/volcanoPlot/VolcanoPlotOpDesc.scala

```diff
@@ -0,0 +1,85 @@
+package edu.uci.ics.amber.operator.visualization.volcanoPlot
+
+import edu.uci.ics.amber.core.tuple.{AttributeType, Schema}
+import edu.uci.ics.amber.core.workflow.OutputPort.OutputMode
+import edu.uci.ics.amber.core.workflow.{InputPort, OutputPort, PortIdentity}
+import edu.uci.ics.amber.operator.PythonOperatorDescriptor
+import edu.uci.ics.amber.operator.metadata.{OperatorGroupConstants, OperatorInfo}
+import com.fasterxml.jackson.annotation.{JsonProperty}
+import com.kjetland.jackson.jsonSchema.annotations.JsonSchemaTitle
+import edu.uci.ics.amber.operator.metadata.annotations.AutofillAttributeName
+
+class VolcanoPlotOpDesc extends PythonOperatorDescriptor {
+
+  @JsonProperty(required = true)
+  @JsonSchemaTitle("Effect Size (log2 Fold Change)")
+  @AutofillAttributeName var effectColumn: String = ""
```

Comment:
Please have a @JsonPropertyDescription here, like we do for all other visualization operators. Without a description  it is very hard to understand what each property does.